src/Microsoft.AspNetCore.Http.Connections/ConnectionsAppBuilderExtensions.cs

```diff
@@ -10,13 +10,13 @@ namespace Microsoft.AspNetCore.Builder
 {
     public static class ConnectionsAppBuilderExtensions
     {
-        public static IApplicationBuilder UseConnections(this IApplicationBuilder app, Action<ConnectionsRouteBuilder> callback)
+        public static IApplicationBuilder UseConnections(this IApplicationBuilder app, Action<ConnectionsRouteBuilder> configure)
```

Comment:
Need to check this is not null

Reply:
`callback = callback ?? throw new ArgumentNullException(nameof(callback));`

@davidfowl 'll love it ;P

Reply:
nooo